cli/internal/runsummary/run_summary.go

```diff
@@ -56,14 +59,15 @@ type Meta struct {
 
 // RunSummary contains a summary of what happens in the `turbo run` command and why.
 type RunSummary struct {
-	ID                ksuid.KSUID        `json:"id"`
-	Version           string             `json:"version"`
-	TurboVersion      string             `json:"turboVersion"`
-	GlobalHashSummary *GlobalHashSummary `json:"globalCacheInputs"`
-	Packages          []string           `json:"packages"`
-	EnvMode           util.EnvMode       `json:"envMode"`
-	ExecutionSummary  *executionSummary  `json:"execution,omitempty"`
-	Tasks             []*TaskSummary     `json:"tasks"`
+	ID                 ksuid.KSUID        `json:"id"`
+	Version            string             `json:"version"`
+	TurboVersion       string             `json:"turboVersion"`
+	GlobalHashSummary  *GlobalHashSummary `json:"globalCacheInputs"`
+	Packages           []string           `json:"packages"`
+	EnvMode            util.EnvMode       `json:"envMode"`
+	FrameworkInference bool               `json:"frameworkInference"`
```

Comment:
Is there any chance we'll add more inference? If so, might makes sense to nest this as `inference.framework`?

Reply:
After how this went I'm expecting it to be extremely limited, and likely opted-into using improved `extends` instead.

Reply:
(We're already discussing switching the default on this flag in the next major, and then I'm going to propose removing the flag + feature in the major after that. But better `extends` is a prerequisite for that.)